app/assets/stylesheets/submission.css

```diff
@@ -13,6 +13,11 @@ table.results {
   vertical-align: top;
 }
 
+.results .inspect_output th {
+  padding: 0;
+  font-size: 0.95em;
```

Comment:
I don't think we should reduce the font size. Word wrapping is inevitable on certain display sizes, and reducing the font size is not a sustainable solution.

Reply:
I'd also prefer keeping it at the standard font size.